projects/ui-particles-angular/src/lib/gio-popover/gio-popover.module.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+
+import { GioIconsModule } from '../gio-icons/gio-icons.module';
+
+import { PopoverTriggerDirective } from './gio-popover-trigger.directive';
+import { GioPopoverComponent } from './gio-popover.component';
+
+@NgModule({
+  imports: [CommonModule, GioIconsModule, MatButtonModule],
+  declarations: [GioPopoverComponent, PopoverTriggerDirective],
```

Comment:
What do you think about keeping the standalone component but using the modules only for import/export as a group utility?
Open question around: "what is the best option now with these standalone components?" 🤔